### PR TITLE
scrub uses of `gw-` in code examples

### PR DESCRIPTION
**Pull request by willbreitkreutz**
I think I got all of them, and there were a few places where the `gw-` got snuck into the wrong spot on some other classes, guess my regex wasn't perfect... 😢

### CODE REVIEW

**Comment by willbreitkreutz** [comment]
I added a rule that requires all pull requests to main be reviewed by someone other than the person submitting it, if it starts to get in the way we can pull it but wanted to try it out.

File: src/app-pages/documentation/forms/color-input.jsx
Comment (jbkolze) [inline src/app-pages/documentation/forms/color-input.jsx:1]: Few more hiding in the Component API block

File: src/app-pages/documentation/forms/date-time-inputs.jsx
Comment (jbkolze) [inline src/app-pages/documentation/forms/date-time-inputs.jsx:1]: Component API block again

File: src/app-pages/documentation/forms/numeric-inputs.jsx
Comment (jbkolze) [inline src/app-pages/documentation/forms/numeric-inputs.jsx:1]: And one more Component API block

**Review by jbkolze — COMMENTED** [review]
Looks good other than a few errors in the input component docs

File: src/app-pages/documentation/forms/color-input.jsx
Comment (willbreitkreutz) [reply src/app-pages/documentation/forms/color-input.jsx:1]: got it

File: src/app-pages/documentation/forms/date-time-inputs.jsx
Comment (willbreitkreutz) [reply src/app-pages/documentation/forms/date-time-inputs.jsx:1]: got it

File: src/app-pages/documentation/forms/numeric-inputs.jsx
Comment (willbreitkreutz) [reply src/app-pages/documentation/forms/numeric-inputs.jsx:1]: got it

**Comment by willbreitkreutz** [comment]
Dang regex went a little crazy, thanks for catching those @jbkolze

**Review by jbkolze — APPROVED** [review]
Looks good @willbreitkreutz -- sorry, honestly meant to approve last time so you wouldn't have to wait on me to approve again

**Comment by willbreitkreutz** [comment]
That's a good question... with a small group i don't really care, but if we see contributions from others then probably a core of "merge doers"

**Pull request merged**